src/formatPkg.js

```diff
@@ -145,13 +146,32 @@ function getLicense(cleaned) {
   return null;
 }
 
-function getOwner(githubRepo, lastPublisher, author) {
-  if (githubRepo) {
-    return {
-      name: githubRepo.user,
-      avatar: `https://github.com/${githubRepo.user}.png`,
-      link: `https://github.com/${githubRepo.user}`,
-    };
+function getOwner(repository, lastPublisher, author) {
+  if (repository && repository.user) {
+    const { user } = repository;
+
+    if (repository.host === 'github.com') {
+      return {
+        name: user,
+        avatar: `https://github.com/${user}.png`,
+        link: `https://github.com/${user}`,
+      };
+    }
+
+    if (repository.host === 'gitlab.com') {
+      return {
+        name: user,
+        link: `https://gitlab.com/${user}`,
```

Comment:
wondering if we should use the npm avatar as a fallback (even though it will usually be wrong)

Reply:
Isn't a bad idea.

Reply:
let's add it so we don't need to add conditionals in the frontend for it